multimodal/src/autogluon/multimodal/models/ft_transformer.py

```diff
@@ -311,6 +311,119 @@ def forward(
         }
 
 
+class AdditiveAttention(nn.Module):
+    """Additive Attention with linear complexity to input sequence length.
+
+    Additive attention was proposed and used in FastFormer.
+    See Ref. [1] for details.
+    This implementation is motivated by: https://github.com/jrzaurin/pytorch-widedeep.git
+
+    References:
+    ----------
+    [1] Wu, Chuhan, et al. "Fastformer: Additive attention can be all you need." arXiv preprint arXiv:2108.09084 (2021).
+    """
+
+    def __init__(
+        self,
+        *,
```

Comment:
Why is there a `*` in the arguments?

Reply:
I noticed MultiHeadAttention did [a similar thing](https://github.com/BingzhaoZhu/autogluon/blob/master/multimodal/src/autogluon/multimodal/models/ft_transformer.py#:~:text=class%20MultiheadAttention(,%2D%3E%20None%3A). Just wish to keep consistent.

Reply:
I see. It forces to use keywords when initializing the class.